### ISSUE TITLE
NTR: Protein Z low in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8386]
New term request Protein Z low in blood (elthat_150807074656): Coagulation downregulator

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000401
- label: Reduced circulating protein Z concentration
- parents: [Abnormal circulating protein concentration (HP:0010876)](https://hpo.jax.org/app/browse/term/HP:0010876)
- pmid: 
- def: The concentration of protein Z in the blood circulation is below the lower limit of normal.
- comment: The vitamin K-dependent protein Z is encoded by the PROZ gene (HGNC:9460).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8386
- synonyms:
